bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,81 @@
+# ABB/Busch-free@home Smart Home binding
+
+ openHAB ABB/Busch-free@home binding based on the offical free@home local api
+
+![alt text](https://github.com/jannodeluxe/jannnnoooo/blob/main/free_at_home_logo_1.jpg)
+![alt text](https://github.com/jannodeluxe/jannnnoooo/blob/main/abb_freeathome_2_0.png)
```

Comment:
Please put any picture into the doc/ directory of your binding to prevent broken links.

Reply:
Fix is done